Packages/com.chisel.components/Chisel/Components/API.public/Components/Base/ChiselBrushContainerAsset.cs

```diff
@@ -5,11 +5,10 @@
 
 namespace Chisel.Components
 {
-    // This is an asset so that when generators share the same output
-    // (when, for example, they're repeated or mirrored) 
-    // they will all automatically update when one is modified.
-
-    // TODO: when not unique, on modification make a copy first and modify that (unless it's an asset in the project?)
+    // This class holds the brushMeshes that have been generated by the generators, 
+    // and holds the instances (not serialized) on the CSG side for these brushMeshes.
+    // This class could be shared between multiple generators that are guaranteed to 
+    // produce the same output. (if for instance, they're copies from each other)
```

Comment:
oops this change should've been in another commit. it simply cleans up things  abit